app/src/main/java/com/xmartlabs/scasas/geoquiz/module/Question.java

```diff
@@ -0,0 +1,46 @@
+package com.xmartlabs.scasas.geoquiz.module;
+
+/**
+ * Created by scasas on 1/25/17.
+ */
+
```

Comment:
If you click on the "light bulb" you can edit the template and remove that space there so you don't have to do it manually